lib/ansible/modules/cloud/azure/azure_rm_virtualmachine.py

```diff
@@ -211,6 +211,7 @@
         choices:
             - Dynamic
             - Static
+            - Disabled
```

Comment:
I can't comment on the line but you should add in the description field a new entry saying `Disabled` was added in Ansible 2.6.